src/index.ts

```diff
@@ -7,6 +7,7 @@ export { Accordion } from './components/Accordion/Accordion'
 export { GovBanner } from './components/GovBanner/GovBanner'
 export { Button } from './components/Button/Button'
 export { ButtonGroup } from './components/ButtonGroup/ButtonGroup'
+// export { Identifier } from './components/Identifier/Identifier'
```

Comment:
This looks like an accident that we can remove

Reply:
Should I remove it or comment it in?

Reply:
Turns out my comment was the 'accident'. I misread which file this was in and just saw a commented line, thinking it should be removed 🤦 

Identifier and any publicly exposed subcomponents _should_ be exported here, so comment it in. I _would_ go ahead give it its own section under a comment though `/** Identifier Components */`